src/commands/create.js

```diff
@@ -146,9 +149,31 @@ export default asyncCommand({
 
 			To start a production HTTP/2 server:
 			  \u001b[32mnpm run serve\u001b[39m
-		`.trim().replace(/^\t+/gm, '') + '\n';
+		`) + '\n';
 	}
 })
 
+const trimLeft = (string) => string.trim().replace(/^\t+/gm, '');
 
 const npm = (cwd, args) => spawn('npm', args, { cwd, stdio: 'ignore' });
+
+// Initializes the folder using `git init` and a proper `.gitignore` file
+// if `git` is present in the $PATH.
+const initializeVersionControl = async function(target) {
+	if (which.sync('git')) {
```

Comment:
Why does this have to be async? It would throw in either case when using async/await.